### PR TITLE
[hi] Localize /en/docs/reference/glossary/disruption.md

### PR DESCRIPTION
**Pull request by surbhiahuja**

### CODE REVIEW

File: content/hi/docs/reference/glossary/disruption.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+title: व्यवधान(Disruption)
+id: व्यवधान(disruption)
```
Comment (sftim) [inline content/hi/docs/reference/glossary/disruption.md:3]:
```suggestion
id: disruption
```

File: content/hi/docs/reference/glossary/disruption.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+फली।
+
+<!--more-->
+
+यदि आप, क्लस्टर ऑपरेटर के रूप में, किसी एप्लिकेशन से संबंधित पॉड को नष्ट करते हैं,
+कुबेरनेट्स का कहना है कि एक _स्वैच्छिक व्यवधान_। यदि कोई पॉड ऑफ़लाइन हो जाता है
+एक नोड विफलता, या एक व्यापक विफलता क्षेत्र को प्रभावित करने वाले आउटेज के कारण,
+कुबेरनेट्स का कहना है कि एक _अनैच्छिक व्यवधान_।
+
+अधिक जानकारी के लिए [व्यवधान] (/docs/concepts/workloads/pods/disruptions/ देखें।
```
Comment (sftim) [inline content/hi/docs/reference/glossary/disruption.md:25]:
```suggestion
अधिक जानकारी के लिए [व्यवधान](/docs/concepts/workloads/pods/disruptions/) देखें।
```

**Review by sftim — COMMENTED** [review]
This won't work with the front matter as proposed. I've suggested a fix.

**Comment by sftim** [comment]
/retitle [hi] Localize glossary entry for “disruption”

**Comment by bishal7679** [comment]
/retitle [hi] Localize /en/docs/reference/glossary/disruption.md

File: content/hi/docs/reference/glossary/disruption.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+---
+title: व्यवधान(Disruption)
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/disruption.md:2]:
```suggestion
title: व्यवधान (Disruption)
```

File: content/hi/docs/reference/glossary/disruption.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+id: disruption
+date: 2019-09-10
+full_link: /docs/concepts/workloads/pods/disruptions/
+short_description: >
+  एक घटना जो पॉड (ओं) को सेवा से बाहर कर देती है
+aka:
+tags:
+- fundamental
+---
+ व्यवधान ऐसी घटनाएँ हैं जो एक या अधिक की ओर ले जाती हैं
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/disruption.md:12]:
```suggestion
व्यवधान ऐसी घटना हैं जो एक या अनेक {{< glossary_tooltip term_id="pod" text="पॉडस " >}} को सेवा से बाहर कर देती हैं। 
```

File: content/hi/docs/reference/glossary/disruption.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
+---
+title: व्यवधान(Disruption)
+id: disruption
+date: 2019-09-10
+full_link: /docs/concepts/workloads/pods/disruptions/
+short_description: >
+  एक घटना जो पॉड (ओं) को सेवा से बाहर कर देती है
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/disruption.md:7]:
```suggestion
  एक घटना जो एक या अनेक पॉड को सेवा से बाहर कर देती हैं। 
```

File: content/hi/docs/reference/glossary/disruption.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+aka:
+tags:
+- fundamental
+---
+ व्यवधान ऐसी घटनाएँ हैं जो एक या अधिक की ओर ले जाती हैं
+{{< glossary_tooltip term_id="pod" text="Pods" >}} सेवा से बाहर हो रहा है।
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/disruption.md:13]:
```suggestion
```

File: content/hi/docs/reference/glossary/disruption.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+कार्यभार संसाधनों के लिए एक व्यवधान के परिणाम होते हैं, जैसे
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/disruption.md:14]:
```suggestion
व्यवधान का कार्यभार संसाधनों पर प्रभाव पड़ता है, जैसे
```

File: content/hi/docs/reference/glossary/disruption.md [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+{{< glossary_tooltip term_id="तैनाती" >}}, जो प्रभावितों पर निर्भर करता है
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/disruption.md:15]:
```suggestion
{{< glossary_tooltip term_id="deployment" text="डिप्लॉयमेंट्स" >}}, जो प्रभावित पॉड्स पर निर्भर करता है। 
```

File: content/hi/docs/reference/glossary/disruption.md [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+फली।
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/disruption.md:16]:
```suggestion
```

File: content/hi/docs/reference/glossary/disruption.md [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+
+<!--अधिक-->
+
+यदि आप, क्लस्टर ऑपरेटर के रूप में, किसी एप्लिकेशन से संबंधित पॉड को नष्ट करते हैं,
+कुबेरनेट्स का कहना है कि एक _स्वैच्छिक व्यवधान_। यदि कोई पॉड ऑफ़लाइन हो जाता है
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/disruption.md:21]:
```suggestion
कुबेरनेट्स के मुताबिक वह  एक _स्वैच्छिक व्यवधान_ हैं  । यदि कोई पॉड ऑफ़लाइन हो जाता है
```

File: content/hi/docs/reference/glossary/disruption.md [hunk 11]
```diff
@@ -0,0 +1,25 @@
...
+एक नोड विफलता, या एक व्यापक विफलता क्षेत्र को प्रभावित करने वाले आउटेज के कारण,
+कुबेरनेट्स का कहना है कि एक _अनैच्छिक व्यवधान_।
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/disruption.md:23]:
```suggestion
कुबेरनेट्स के मुताबिक वह एक _अनैच्छिक व्यवधान_ हैं ।
```

File: content/hi/docs/reference/glossary/disruption.md (hunk 2 above)
Comment (divya-mohan0209) [reply content/hi/docs/reference/glossary/disruption.md:25]: Please address this comment @surbhiahuja

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]
Please address these comments

File: content/hi/docs/reference/glossary/disruption.md [hunk 12]
```diff
@@ -0,0 +1,25 @@
...
+id: disruption
+date: 2019-09-10
+full_link: /docs/concepts/workloads/pods/disruptions/
+short_description: >
+  एक घटना जो एक या अनेक पॉड को सेवा से बाहर कर देती हैं।
+aka:
+tags:
+- fundamental
+---
+ व्यवधान ऐसी घटना हैं जो एक या अनेक {{< glossary_tooltip term_id="pod" text="पॉडस " >}} को सेवा से बाहर कर देती हैं।
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/disruption.md:12]:
```suggestion
 व्यवधान ऐसी घटना हैं जो एक या अनेक {{< glossary_tooltip term_id="pod" text="पॉड्स" >}} को सेवा से बाहर कर देती हैं।
```

File: content/hi/docs/reference/glossary/disruption.md [hunk 13]
```diff
@@ -0,0 +1,25 @@
...
+
+व्यवधान का कार्यभार संसाधनों पर प्रभाव पड़ता है, जैसे
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/disruption.md:14]:
```suggestion
व्यवधान कार्यभार संसाधन को प्रभावित करते हैं, जैसे
```

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]
1. Amend commit message
2. Suggestions left from review

**Comment by Babapool** [comment]
Hi @surbhiahuja , just a gentle ping, since there has been no updates on this PR for a long time are you still working on this PR? Thanks

**Comment by bishal7679** [comment]
!!GENTLE REMINDER!!
@surbhiahuja Feel free to let us know whether you are actively working on this PR and commit the above suggestion
or we'll have to assign this to another interested contributor

**Comment by divya-mohan0209** [comment]
Hello @surbhiahuja , since there has been no response/activity on the PR since quite some time, I'll be closing it off. Please feel free to reopen it, whenever you have the bandwidth to commence work on it.
/close

**Pull request closed without merging**